src/crimson/os/alienstore/alien_store.cc

```diff
@@ -76,14 +76,15 @@ AlienStore::AlienStore(const std::string& type,
   : type(type),
     path{path}
 {
-  cct = std::make_unique<CephContext>(CEPH_ENTITY_TYPE_OSD);
-  g_ceph_context = cct.get();
-  cct->_conf.set_config_values(values);
```

Comment:
isn't this line required anymore?

Reply:
I reworded commit message. Please check if it now makes sense.